lib/oidc/handleOAuthResponse.ts

```diff
@@ -112,7 +113,7 @@ export function handleOAuthResponse(sdk: OktaAuth, tokenParams: TokenParams, res
         var idTokenObj: IDToken = {
           idToken: idToken,
           claims: idJwt.payload,
-          expiresAt: idJwt.payload.exp,
+          expiresAt: idJwt.payload.exp - idJwt.payload.iat + now,
```

Comment:
add a small comment explaining what is happening here? (adjusting expiresAt to be in local time)